src/storage-client/src/controller/rehydration.rs

```diff
@@ -68,6 +69,7 @@ where
         build_info: &'static BuildInfo,
         metrics: RehydratingStorageClientMetrics,
         envd_epoch: NonZeroI64,
+        grpc_client_params: GrpcClientParameters,
```

Comment:
these can only change on restart? (i think thats good, just wondering)

Reply:
yes -- it'll only change on restart per replica